.mkosi/mkosi.fedora

```diff
@@ -45,6 +45,7 @@ BuildPackages=
         gnu-efi-devel
         gnutls-devel
         gperf
+        initscripts
```

Comment:
so, i am tempted to declare sushell a fedora-ism, and we should not make use of it at all, but instead just use /bin/sh by default, with a configure time option to use something else. Then, for the mkosi invocations we won't bother, and get /bin/sh as before, but for fedora this can be redirected to sushell. will fix